torch/_inductor/codegen/aoti_runtime/interface.cpp

```diff
@@ -26,6 +26,17 @@
             std::to_string(actual_size));                         \
   } while (0)
 
+// AOTInductor uses at::addmm_out, which doesn't supports
+// arguments that requires gradient. For this reason, we
+// enforce no_grad context for run APIs.
+#define WITH_NO_GRAD(...)                               \
+  do {                                                  \
+    bool prev_mode = aoti_torch_grad_mode_is_enabled(); \
```

Comment:
Consider print a warning msg if `prev_mode` is true.

Reply:
This will be invoked in critical path of run(), so I shouldn't print log at every run() call. 

I can print warning at ContainerCreate(), but there is a chance that create() is called under no_grad, but run() is not.